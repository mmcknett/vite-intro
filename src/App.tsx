import { useState } from 'react';
import reactLogo from './assets/react.svg';
import './App.css';
import Turn from './Turn';

function App() {
  const [count, setCount] = useState(0)

  const getText = () => 'Click on the Vite and React logos to learn more!';

  return (
    <div className="App">
      <h1>Vordle</h1>
      <h2>Example</h2>
      <p>
        <span style={{ color: "green" }}>A</span>UDIO <br/>
        <span style={{ color: "green" }}>A</span>NGL<span style={{ color: "yellow" }}>E</span> <br/>
        ...<br/>
        <span style={{ color: "green" }}>AMBER</span><br/>
        <Turn guess="ANGLE" referenceWord='AMBER' />
      </p>
      <div className="card">
        <input type="text" />
        <button onClick={ () => {}}>Guess</button>
      </div>
      <p className="read-the-docs">
        { getText() }
      </p>
    </div>
  )
}

export default App
