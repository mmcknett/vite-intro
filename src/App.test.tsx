import { describe, it, expect } from 'vitest';
import { render } from '@testing-library/react';
import App from './App.jsx';

describe('App', () => {
  it('renders with no props as expected', () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
  })
});