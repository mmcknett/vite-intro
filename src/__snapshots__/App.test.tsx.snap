// Vitest Snapshot v1

exports[`App > renders with no props as expected 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div>
      <a
        href="https://vitejs.dev"
        target="_blank"
      >
        <img
          alt="Vite logo"
          class="logo"
          src="/vite.svg"
        />
      </a>
      <a
        href="https://reactjs.org"
        target="_blank"
      >
        <img
          alt="React logo"
          class="logo react"
          src="/src/assets/react.svg"
        />
      </a>
    </div>
    <h1>
      Vite + React
    </h1>
    <div
      class="card"
    >
      <button>
        count is 0
      </button>
      <p>
        Edit 
        <code>
          src/App.tsx
        </code>
         and save to test HMR
      </p>
    </div>
    <p
      class="read-the-docs"
    >
      Click on the Vite and React logos to learn more!
    </p>
  </div>
</DocumentFragment>
`;
